lib/absinthe_error_payload/changeset_parser.ex

```diff
@@ -139,6 +139,9 @@ defmodule AbsintheErrorPayload.ChangesetParser do
 
   defp interpolated_value_to_string(value) when is_list(value), do: Enum.join(value, ",")
 
+  defp interpolated_value_to_string({:parameterized, Ecto.Enum, %{on_load: mappings}}),
+    do: Map.values(mappings) |> Enum.join(",")
```

Comment:
This should be written like this to pass Credo checks:

```suggestion
  defp interpolated_value_to_string({:parameterized, Ecto.Enum, %{on_load: mappings}}),
    do: mappings |> Map.values() |> Enum.join(",")
```